src/Illuminate/Foundation/Application.php

```diff
@@ -1192,6 +1202,19 @@ public function setLocale($locale)
         $this['events']->dispatch(new LocaleUpdated($locale));
     }
 
+    /**
+     * Set the current application fallback locale.
+     *
+     * @param  string  $fallbackLocale
+     * @return void
+     */
+    public function setFallbackLocale($fallbackLocale)
+    {
+        $this['config']->set('app.fallback_locale', $fallbackLocale);
+
+        $this['translator']->setFallback($fallbackLocale);
```

Comment:
This force instantiates the translator. You should check if it has been created yet.